src/renderers/dom/client/wrappers/__tests__/ReactDOMTextarea-test.js

```diff
@@ -163,6 +171,57 @@ describe('ReactDOMTextarea', function() {
     expect(node.value).toEqual('foo');
   });
 
+  it('should take updates to `defaultValue` for uncontrolled textarea', function() {
+    var container = document.createElement('div');
+
+    var el = ReactDOM.render(<textarea type="text" defaultValue="0" />, container);
+    var node = ReactDOM.findDOMNode(el);
+
+    expect(node.value).toBe('0');
+
+    ReactDOM.render(<textarea type="text" defaultValue="1" />, container);
+
+    expect(node.value).toBe('1');
+  });
+
+  it('should take updates to children in lieu of `defaultValue` for uncontrolled textarea', function() {
+    var container = document.createElement('div');
+
+    var el = ReactDOM.render(<textarea type="text" defaultValue="0" />, container);
+    var node = ReactDOM.findDOMNode(el);
```

Comment:
Same, no need for these findDOMNode calls, `ReactDOM.render` returns the DOM node.